runtime/port/unix/j9sysinfo.c

```diff
@@ -66,10 +66,10 @@
 #include "j9csrsi.h"
 #endif /* defined(J9ZOS390) */
 
-#if defined(LINUXPPC)
+#if (defined(LINUXPPC) || (defined(S390) && !defined(J9ZOS390)))
```

Comment:
Are we able to turn this negative statement into a positive one using `defined(LINUX)`? i.e.

```
if (defined(LINUXPPC) || (defined(S390) && defined(LINUX)))
```

Reply:
what do we have set for ZTPF? It is "LINUX" for sure... Does it have "S390" set as well?

Reply:
@pelemie could you confirm what we have to ZTPF?

Reply:
how would auxv work on ZTPF? Could it be the right thing to do to run auxv the check?

Reply:
I changed it to if (defined(LINUXPPC) || (defined(S390) && defined(LINUX))). If it is true for ZTPF would that be an issue or something we actually want to be true for ZTPF?

Reply:
> I changed it to if (defined(LINUXPPC) || (defined(S390) && defined(LINUX))). If it is true for ZTPF would that be an issue or something we actually want to be true for ZTPF?

I'm not really sure. ZTPF is not a supported platform in OpenJ9 so there is no way for us to test either way. I think what you have is fine.